client/blocks/image-editor/index.jsx

```diff
@@ -51,8 +51,9 @@ const ImageEditor = React.createClass( {
 	getDefaultProps() {
 		return {
 			media: null,
-			onImageEditorSave: noop,
-			onImageEditorCancel: noop
+			onImageExtracted: noop,
+			onCancel: null,
```

Comment:
I thought the commit description explained this well :(

![selection_024](https://cloud.githubusercontent.com/assets/4988512/19366092/0b139550-9196-11e6-97dd-4a7ac4c718df.jpg)